tools/downloader/common.py

```diff
@@ -80,6 +80,7 @@
 }
 
 KNOWN_QUANTIZED_PRECISIONS = {p + '-INT8': p for p in ['FP16', 'FP32']}
+KNOWN_COMPILABLE_PRECISIONS = {'FP16', 'FP32'}
```

Comment:
This variable isn't used outside the compiler script, so it can live there.

Also, why only these precisions? Can quantized models not be compiled?

Reply:
Compiler doesn't support quantized models, as I understand. It shows errors about `unsupported layer type "FakeQuantize"`

Reply:
Moved `KNOWN_COMPILABLE_PRECISIONS` to the compiler script.